packages/@aws-cdk/aws-pipes-enrichments-alpha/README.md

```diff
@@ -88,3 +88,22 @@ const pipe = new pipes.Pipe(this, 'Pipe', {
   target: new SomeTarget(targetQueue),
 });
 ```
+
+### API Gateway (REST API)
+
+API Gateway can be used to enrich events of a pipe.
```

Comment:
Thanks for the PR @mazyu36 - I'm curious if you've considered ApiGatewayV2; whether or not the enrichment should cover that case as well, and if so whether that would be a separate construct or also handled by this design?

Reply:
@sumupitchayan 

ApigatewayV2(HTTP API) cannot be used because pipes only supports Apigateway(REST API).

https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-aws-services-reference.html

The pipes documentation did not clearly state this.
Also, when I tried in the management console, I was unable to use the HTTP API.​​​​​​​​​​​​​​​​

Reply:
Got it. Can you maybe then also make this more clear in the README?

Reply:
Thanks. I've updated README.